src/app/components/taxonomy/TaxonomyDashboard.jsx

```diff
@@ -35,6 +36,31 @@ const selectTaxonomyTables = f.compose(
   f.propOr([], "tables.data")
 );
 
+const TaxonomyTiles = ({ tables, langtag }) => (
+  <section className="taxonomy-dashboard__content">
+    {tables.map(table => (
+      <TaxonomyTableCard key={table.name} table={table} langtag={langtag} />
+    ))}
+  </section>
+);
+
+const NoTaxonomiesYet = () => (
+  <section className="taxonomy-dashboard__content--empty">
+    <img className="fancy-image" alt="" src="/img/taxonomies-empty.png" />
+    <div className="title">{i18n.t("dashboard:taxonomy.title")}</div>
+    <div className="description">
+      {i18n.t("dashboard:taxonomy.description")}
+    </div>
+
+    <a
+      className="support-email-button"
+      href={`mailto://${supportDetails.email}`}
```

Comment:
Hier sind die // zu viel.
```suggestion
      href={`mailto:${supportDetails.email}`}
```